examples/sticky.ads.amp.html

```diff
@@ -160,7 +159,7 @@
     </div>
   </header>
   <main role="main">
-  <amp-sticky-ad layout="nodisplay">
+  <amp-sticky-ad layout="container">
```

Comment:
Do you mean that before we get `amp-sticky-ad` css file, it might be treated as a normal div and would be visible. 
changed the layout type back to `nodisplay`